scripts/foundry/contract-upgrades/2.1.0/README.md

```diff
@@ -0,0 +1,51 @@
+# Nitro contracts 2.1.0 upgrade
+These scripts empower `NitroContracts2Point1Point0UpgradeAction` action contract which performs upgrade to [2.1.0 release](https://github.com/OffchainLabs/nitro-contracts/releases/tag/v2.1.0) of Nitro contracts for existing Orbit chains. Predeployed instances of the upgrade action exists on the chains listed in the following section with vanilla ArbOS31 wasm module root set. If you have a custom nitro machine, you will need to deploy the upgrade action yourself.
+
+NitroContracts2Point1Point0UpgradeAction will perform the following action:
+1. Upgrade ChallengeManager to v2.1.0
+2. Set the wasm module root to the new version
+3. Set the conditional wasm root and one step proof
+4. Upgrade RollupAdminLogic and RollupUserLogic contracts
+
+Note that contracts without code changes are not upgraded. It is normal to have some contracts still in the old version after the upgrade as they are equivalent to the new version. After the contract upgrade, you would need to schedule an ArbOS upgrade to ArbOS31 to enable the new features.
+
+## Requirements
+This upgrade only support upgrading from the following [nitro-contract release](https://github.com/OffchainLabs/nitro-contracts/releases):
+- Inbox: v1.1.0 - v2.1.0 inclusive
+- Outbox: v1.1.0 - v2.1.0 inclusive
+- SequencerInbox: v1.2.1 - v2.1.0 inclusive
+- Bridge: v1.1.0 - v2.1.0 inclusive
+- RollupProxy: v1.1.0 - v2.1.0 inclusive
+- RollupAdminLogic: v1.1.0 - v2.1.0 inclusive
+- RollupUserLogic: v1.1.0 - v2.1.0 inclusive
+- ChallengeManager: v1.2.1 - v1.3.0 inclusive
```

Comment:
```suggestion
- Inbox: v1.1.0 - v1.3.0 inclusive
- Outbox: v1.1.0 - v1.3.0 inclusive
- SequencerInbox: v1.2.1 - v2.1.0 inclusive
- Bridge: v1.1.0 - v1.3.0 inclusive
- RollupProxy: v1.1.0 - v2.1.0 inclusive
- RollupAdminLogic: v1.1.0 - v1.3.0 inclusive
- RollupUserLogic: v1.1.0 - v1.3.0 inclusive
- ChallengeManager: v1.2.1 - v1.3.0 inclusive
```
Let's do this instead. In general, any contracts that this upgrade touched and those not touched but with storage change should be excluded. Also update the versioner to align.

Reply:
https://github.com/OffchainLabs/orbit-actions/pull/16/commits/07870dcdbf4405d164f5332039b93a151fa2fcc0